<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>favorite</class>
 <widget class="QWidget" name="favorite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/new/prefix1/src/videoplayer.png</normaloff>:/new/prefix1/src/videoplayer.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>90</width>
        <height>90</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="image.qrc">:/new/prefix1/src/collect.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>17</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>收藏夹列表</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>你可以编辑、删除、排序或添加新的项目,双击其单元格就可以编辑其内容。</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>275</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>275</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>541</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>172</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_Ok">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>534</width>
        <height>73</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>150</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>12</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_AddNew">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>新建项目(&amp;N)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_Play">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>131</width>
             <height>26</height>
            </size>
           </property>
           <property name="text">
            <string>播放(&amp;S)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>40</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_Del">
             <property name="minimumSize">
              <size>
               <width>115</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>115</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>删除(&amp;E)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_DelAll">
             <property name="minimumSize">
              <size>
               <width>115</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>115</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>删除全部(&amp;A)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QPushButton" name="btn_Up">
             <property name="minimumSize">
              <size>
               <width>88</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>88</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>向上(&amp;U)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_Down">
             <property name="minimumSize">
              <size>
               <width>88</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>88</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string>向下(&amp;D)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
